src/wp-includes/media.php

```diff
@@ -2349,6 +2349,12 @@ function _wp_post_thumbnail_context_filter_remove() {
  * @return string HTML content to display the caption.
  */
 function img_caption_shortcode( $attr, $content = '' ) {
+	// validate input parameter type for compatibility with newer versions of PHP
+	// (this maintains behavior consistent with how this function worked on 
+	// PHP versions 7.0 and older.)
```

Comment:
```suggestion
	/*
	 * validate input parameter type for compatibility with newer versions of PHP
	 * (this maintains behavior consistent with how this function worked on 
	 * PHP versions 7.0 and older.)
	 */
```
Use multiline comment.